### PR TITLE
Mon 15260 visual bugs within new theme

### PR DESCRIPTION
**Pull request by PascalTassel**
## Description

Correction of few visual bugs that have been identified concerning the new 22.10 theme.

- A resizing in the header was triggered in Status details pages when doing the following sequence: pause / play / pause.
- Menu items contrast in dark mode.
- Remove tiles inner borders in the Resource status panel.
- Adjustment of spacing around buttons in most configuration pages.
- Reduce indicator size in the ressource status table.
- Reduce indicator size in the header of ressource status panel.
- Change variant of the button in timeline tab of ressource status panel, replacing primary with text
- Justify date to the right in header
- Change the color of the separation line by the color Divider. 

screenshot

[recording-2022-10-05-17-54-53.webm](https://user-images.githubusercontent.com/51056870/194818709-72cc853e-9a07-4529-b23b-d60cd961cb11.webm)

## Type of change

- [X] Patch fixing an issue (non-breaking change)
- [ ] New functionality (non-breaking change)
- [ ] Breaking change (patch or feature) that might cause side effects breaking part of the Software

## Target serie

- [ ] 21.04.x
- [ ] 21.10.x
- [ ] 22.04.x
- [X] 22.10.x (master)

### CODE REVIEW

**Review by Thebarda — APPROVED** [review]
Ok for the react part

File: features/bootstrap/VirtualMetricHandleContext.php [hunk 1]
```diff
@@ -20,7 +20,7 @@ class VirtualMetricHandleContext extends CentreonContext
     public function iAddAVirtualMetric()
     {
         $this->page = new MetricsConfigurationListingPage($this);
-        $this->assertFind('css', 'a[class="btc bt_success"]')->click();
+        $this->assertFind('css', 'a[class="btc bt_success ms-1"]')->click();
```
Comment (JKancel) [inline features/bootstrap/VirtualMetricHandleContext.php]: I see your question from the daily meeting. In that case isn't it better to replace `class=` by `class*=` (see the suggestion) in order to make the assert more robust ?
```suggestion
        $this->assertFind('css', 'a[class*="btc bt_success"]')->click();
```

File: www/Themes/Generic-theme/style.css [hunk 2]
```diff
@@ -2954,3 +2956,14 @@ ul.module_list {
     background-color: var(--gorgone-config-file-background-color);
 }
 
+/* utilities */
+.ms-1 {
+    margin-left: 8px;
+}
+.me-1 {
```
Comment (JKancel) [inline www/Themes/Generic-theme/style.css]: Why not `ml` for margin-left and `mr` for margin-right ?

File: features/bootstrap/VirtualMetricHandleContext.php (hunk 1 above)
Comment (PascalTassel) [reply features/bootstrap/VirtualMetricHandleContext.php]: You're right Joffrey, I correct !

File: www/Themes/Generic-theme/style.css (hunk 2 above)
Comment (PascalTassel) [reply www/Themes/Generic-theme/style.css]: s as "start", e as "end". Inspired by Bootstrap 5.

Comment (PascalTassel) [reply www/Themes/Generic-theme/style.css]: I replaced the classes name with yours:
- replaced `ms-1` with `ml-1`
- replaced `me-1` with `mr-1`

**Review by bdauria — APPROVED** [review]

**Review by JKancel — APPROVED** [review]

**Review by kduret — APPROVED** [review]

**Pull request merged**